packages/client/src/contents/getSEOFiles.ts

```diff
@@ -0,0 +1,19 @@
+import { adaptError } from '../helpers/client/formatError.js';
+import client from '../helpers/client/index.js';
+import flattenObject from '../utils/flattenObject.js';
+import join from 'proper-url-join';
+import type { Config } from '../types/index.js';
+import type { GetSEOFilesQuery, SEOFiles } from './types/seoFiles.types.js';
+
+const getSEOFiles = (
+  query: GetSEOFilesQuery,
+  config?: Config,
+): Promise<SEOFiles> =>
+  client
+    .get(join('/content/v1/seofiles', { query: flattenObject(query) }), config)
```

Comment:
Looking at the query type, the `flattenObject` call is unnecessary.

```suggestion
    .get(join('/content/v1/seofiles', { query }), config)
```

Reply:
Resolved! ✅